install.sh

```diff
@@ -55,7 +55,11 @@ do_download ${URL} ${ZIP_PATH}
 
 echo "Installing summon v${LATEST_VERSION} into /usr/local/bin"
 
-sudo tar -C /usr/local/bin -zxvf ${ZIP_PATH}
+if sudo 2>/dev/null; then
```

Comment:
on my machine it's never hitting this case, because `sudo` by itself always exits with a nonzero code and returns the help content.

I would suggest updating this to `sudo -h > /dev/null` to achieve the functionality you desire (which it seems is to run as sudo when sudo is available, and without sudo if not)